container/common/container_hints.go

```diff
@@ -25,7 +25,7 @@ import (
 	"os"
 )
 
-var argContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
+var ArgContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
```

Comment:
I wasn't saying it was a test flag, I was saying we couldn't make the function not take an argument, but just use the flags value as test code passes a value to the function